kotsadm/pkg/apiserver/bootstrap.go

```diff
@@ -0,0 +1,57 @@
+package apiserver
+
+import (
+	"log"
+	"os"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	awssession "github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/pkg/errors"
+	kotss3 "github.com/replicatedhq/kots/kotsadm/pkg/s3"
+)
+
+func bootstrap() error {
+	if err := ensureS3Bucket(); err != nil {
+		return errors.Wrap(err, "failed to ensure s3 bucket")
+	}
+
+	return nil
+}
+
+func ensureS3Bucket() error {
```

Comment:
Can this be in the S3PGStore, in an init function?  Not all stored need this.

Reply:
yeah...